twisted/cred/checkers.py

```diff
@@ -76,7 +76,7 @@ class InMemoryUsernamePasswordDatabaseDontUse(object):
                             credentials.IUsernameHashedPassword)
 
     def __init__(self, **users):
-        self.users = users
+        self.users = {x.encode('ascii'):y for x, y in users.items()}
```

Comment:
minor comment:

why not encode with utf-8 ? ... so that we can also use this code to test unicode related usernames / passwords ?